cmd/cstor-pool-mgmt/controller/common/common.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2018 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/openebs/maya/cmd/cstor-pool-mgmt/pool"
+	clientset "github.com/openebs/maya/pkg/client/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CStor controllers to be watched
+const (
+	CStorPool          = "cStorPool"
+	CStorVolumeReplica = "cStorVolumeReplica"
+)
+
+const (
+	// SuccessSynced is used as part of the Event 'reason' when a resource is synced
+	SuccessSynced = "Synced"
+	// ErrResourceExists is used as part of the Event 'reason' when a resource fails
+	// to sync due to a resource of the same name already existing.
+	ErrResourceExists = "ErrResourceExists"
+
+	// MessageResourceExists is the message used for Events which
+	// fails to sync due to a resource already existing
+	MessageResourceExists = "Resource %q already exists and cannot be handled"
```

Comment:
The message seems to imply that the resource cannot be handled. If its the event you are talking about, then there should be better description of the error message.

Reply:
will look into this.